code/modules/customitems/item_defines.dm

```diff
@@ -1,753 +0,0 @@
-	name = "Nashida's Etched Beaker"
-	desc = "The message: 'Please do not be removing this beaker from the chemistry lab. If lost, return to Nashida Bisha'ra' can be seen etched into the side of this 100 unit beaker."
-	icon = 'icons/obj/chemical.dmi'
-	icon_state = "beakerlarge"
-	g_amt = 5000
-	volume = 100
-
-/obj/item/weapon/storage/pill_bottle/fluff/listermedbottle //compactninja: Lister Black
-	name = "Pill bottle (anti-depressants)"
-	desc = "Contains pills used to deal with depression. They appear to be prescribed to Lister Black"
-	New()
-		..()
-		new /obj/item/weapon/reagent_containers/pill/fluff/listermed( src )
-		new /obj/item/weapon/reagent_containers/pill/fluff/listermed( src )
-		new /obj/item/weapon/reagent_containers/pill/fluff/listermed( src )
-		new /obj/item/weapon/reagent_containers/pill/fluff/listermed( src )
-		new /obj/item/weapon/reagent_containers/pill/fluff/listermed( src )
-		new /obj/item/weapon/reagent_containers/pill/fluff/listermed( src )
-		new /obj/item/weapon/reagent_containers/pill/fluff/listermed( src )
-
-/obj/item/weapon/reagent_containers/pill/fluff/listermed
-	name = "anti-depressant pill"
-	desc = "Used to deal with depression."
-	icon_state = "pill10" //white round
-	New()
-		..()
-		reagents.add_reagent(STOXIN, 5)
-		reagents.add_reagent(SUGAR, 10)
-		reagents.add_reagent(ETHANOL, 5)
-
-/obj/item/clothing/mask/fluff/electriccig //CubeJackal: Barry Sharke
-	name = "Electronic cigarette"
-	desc = "An electronic cigarette. Most of the relief of a real cigarette with none of the side effects. Often used by smokers who are trying to quit the habit."
-	icon = 'icons/obj/custom_items.dmi'
-	icon_state = "cigon"
-	throw_speed = 0.5
-	item_state = "ciglit"
-	w_class = W_CLASS_TINY
-	body_parts_covered = null
-	flags = FPRINT|TABLEPASS
-
-//Strange penlight, Nerezza: Asher Spock
-
-/obj/item/weapon/reagent_containers/hypospray/fluff/asher_spock_1
-	name = "strange penlight"
-	desc = "Besides the coloring, this penlight looks rather normal and innocent. However, you get a nagging feeling whenever you see it..."
-	icon = 'icons/obj/custom_items.dmi'
-	icon_state = "asher_spock_1"
-	amount_per_transfer_from_this = 5
-	volume = 15
-
-/obj/item/weapon/reagent_containers/hypospray/fluff/asher_spock_1/New()
-	..()
-	reagents.remove_reagent(TRICORDRAZINE, 30)
-	reagents.add_reagent(OXYCODONE, 15)
-	update_icon()
-	return
-
-/obj/item/weapon/reagent_containers/hypospray/fluff/asher_spock_1/attack_self(mob/user as mob)
-	to_chat(user, "<span class='notice'>You click \the [src] but get no reaction. Must be dead.</span>")
-
-/obj/item/weapon/reagent_containers/hypospray/fluff/asher_spock_1/attack(mob/M as mob, mob/user as mob)
-	if (user.ckey != "nerezza") //Because this can end up in the wrong hands, let's make it useless for them!
-		to_chat(user, "<span class='notice'>You click \the [src] but get no reaction. Must be dead.</span>")
```

Comment:
This was the one that I remembered this morning and made me track this file down.